sdktests/server_side_eval.go

```diff
@@ -78,31 +78,44 @@ func RunParameterizedServerSideEvalTests(t *ldtest.T) {
 }
 
 func RunParameterizedServerSideClientNotReadyEvalTests(t *ldtest.T) {
+	defaultValues := DefaultValueByTypeFactory()
+	flagKey := "some-flag"
+	user := lduser.NewUser("user-key")
+	expectedReason := ldreason.NewEvalReasonError(ldreason.EvalErrorClientNotReady)
+
 	dataSource := NewSDKDataSource(t, mockld.BlockingUnavailableSDKData(mockld.ServerSideSDK))
 	client := NewSDKClient(t,
 		WithConfig(servicedef.SDKConfigParams{StartWaitTimeMS: 1, TimeoutOK: true}),
 		dataSource)
 
-	for _, suite := range getAllServerSideEvalTestSuites(t, "server-side-eval-client-not-ready") {
```

Comment:
There wasn't really a good reason to use the parameterized data file system for this test, because the only parameter is the value type, and we already have a fairly convenient way to do that programmatically.